spec/models/tag_spec.rb

```diff
@@ -20,6 +20,31 @@
       expect(bad_tag.posts.count).to eq(0)
       expect(good_tag.posts.count).to eq(5)
     end
+
+    it "takes the correct actions for user tags" do
+      good_tag = create(:content_warning)
+      bad_tag = create(:content_warning)
+
+      user1 = create(:user)
+      user1.update!(content_warning_ids: [good_tag.id])
+      user2 = create(:user)
+      user2.update!(content_warning_ids: [good_tag.id])
+      user3 = create(:user)
+      user3.update!(content_warning_ids: [good_tag.id])
+      user4 = create(:user)
+      user4.update!(content_warning_ids: [bad_tag.id])
+      user5 = create(:user)
+      user5.update!(content_warning_ids: [bad_tag.id])
```

Comment:
I think you could write this:

```suggestion
      user1, user2, user3 = create_list(:user, 3, content_warning_ids: [good_tag.id])
      user4, user5 = create_list(:user, 2, content_warning_ids: [bad_tag.id])
```